examples/src/agents/zapier_mrkl.ts

```diff
@@ -0,0 +1,25 @@
+import { OpenAI } from "langchain";
+import { initializeAgentExecutor, ZapierToolKit } from "langchain/agents";
+import { ZapierNLAWrapper } from "langchain/zapier";
```

Comment:
Do we really want to have an entrypoint dedicated to zapier? Why not stick this in tools or something like that

Reply:
did a small refactor